src/components/DatasetPreviewView.jsx

```diff
@@ -155,12 +155,12 @@ export class DatasetPreviewView extends React.PureComponent {
                 </div>
               ) : (
                 <div className={classes.values}>
-                  {createdDataset && createdDataset.dataset.name}
+                  {createdDataset && createdDataset.datasetRequest.name}
```

Comment:
do we need to check createdDataset.datasetRequest here and on line 163?

Reply:
hmmmm--yes. It feels more likely that createdDataset would be the thing that gets lost, but no reason not to have the extra safety rails